recipes/nullarbor/meta.yaml

```diff
@@ -1,70 +1,67 @@
 {% set name = "nullarbor" %}
-{% set version = "2.0.20181010" %}
+{% set version = "2.0.20191003" %}
+{% set sha256 = "85dd1db79de41658ad5eb9934b85beb559278e42e60ccd8cda1933bab4d84792" %}
+{% set user = "tseemann" %}
 
 package:
-  name: nullarbor
+  name: {{ name }}
   version: {{ version }}
 
-build:
-  number: 5
-  skip: True # [osx]
-
 source:
-  git_url: https://github.com/tseemann/nullarbor.git
-  git_rev: d561e9d
-  md5: unused
+  url: https://github.com/{{ user }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
 
 requirements:
   run:
-    - perl
```

Comment:
please leave perl in

Reply:
ok, i assumed the perl mod deps would take care of that. but i will add back.

Reply:
This is technically true. But we use those explicit language packages to determine against how many different versions we need to build the package. This is more important for python/ruby/R, but hopefully someone will care about the perl ecosystem at some point and then its good to have this in place - to be consistent.